.github/workflows/deploy-examples.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy example apps to Fly
+
+on:
+  # Allow calling this workflow from other workflows
+  workflow_call:
+    inputs:
+      version:
+        type: string
```

Comment:
Are there any other input types (e.g., multiple choice based on releases) or is this just wishful thinking?

Reply:
Only string, number, and boolean